apps/financial-aid/api/src/app/modules/application/dto/applications.input.ts

```diff
@@ -0,0 +1,11 @@
+import { Allow } from 'class-validator'
+
+import { Field, InputType } from '@nestjs/graphql'
+import { ApplicationStateUrl } from '@island.is/financial-aid/shared/lib'
+
+@InputType()
+export class ApplicationsInput {
```

Comment:
Should we change this to AllApplicationInput, so we don't mix it with ApplicationInput